### DISCUSSION TITLE
请问是否可以用Enjoy学习日语

### DISCUSSION THREAD

**Discussion by NeoLyu** [ZuodaoTech/everyone-can-use-english#708] [Q&A]
如题，如果可以的话，是不是需要额外购买GPT4的token，感激不尽。

**Comment by zlcolin** [comment]
同gpt4 token无直接关系，同模型有关，比如你选择支持日文多语言模型就可以
https://github.com/openai/whisper

**Comment by an-lee** [comment]
可以的。日语材料需要自己准备，导入现有的日语音频，或者自己用智能助手生成（可以参考 Enjoy 里预设的“英语教练”角色）。
不需要额外准备 OpenAI 的 key，直接在 Enjoy 里充值就可以使用了。
